api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -28,14 +28,85 @@
 		See http://www.liquibase.org/manual/home#available_database_refactorings
 		for a list of supported elements and attributes
 	-->
+	<changeSet id="TRUNK-5410" author="tendomart">
+		<comment>Creating order_group_attribute_type table</comment>
+		<createTable tableName="order_group_attribute_type">
+			<column name="order_group_attribute_type_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="name" type="varchar(255)">
+				<constraints nullable="false" unique="true"/>
+			</column>
+			<column name="description" type="varchar(1024)" />
+			<column name="datatype" type="varchar(255)" />
+			<column name="datatype_config" type="text" />
+			<column name="preferred_handler" type="varchar(255)" />
+			<column name="handler_config" type="text" />
+			<column name="min_occurs" type="int">
+				<constraints nullable="false" />
+			</column>
+			<column name="max_occurs" type="int"></column>
+			<column name="creator" type="int">
+				<constraints nullable="false" />
+			</column>
+			<column name="date_created" type="datetime">
+				<constraints nullable="false" />
+			</column>
+			<column name="changed_by" type="int" />
+			<column name="date_changed" type="datetime" />
+			<column name="retired" type="boolean" defaultValueBoolean="false">
+				<constraints nullable="false" />
+			</column>
+			<column name="retired_by" type="int" />
+			<column name="date_retired" type="datetime" />
+			<column name="retire_reason" type="varchar(255)" defaultValue="null" />
+			<column name="uuid" type="char(38)">
+				<constraints nullable="false" unique="true" />
+			</column>
+		</createTable>
+		<addForeignKeyConstraint constraintName="order_group_attribute_type_creator_fk" baseTableName="order_group_attribute_type" baseColumnNames="creator" referencedTableName="users" referencedColumnNames="user_id"/>
+		<addForeignKeyConstraint constraintName="order_group_attribute_type_changed_by_fk" baseTableName="order_group_attribute_type" baseColumnNames="changed_by" referencedTableName="users" referencedColumnNames="user_id"/>
+		<addForeignKeyConstraint constraintName="order_group_attribute_type_retired_by_fk" baseTableName="order_group_attribute_type" baseColumnNames="retired_by" referencedTableName="users" referencedColumnNames="user_id"/>
+		<modifySql dbms="mssql">
```

Comment:
What are we using this for?

Reply:
@dkayiwa  you mean the  <**modifySql** dbms="mssql"> tag alone

or rather the **addForeignKeyConstraint** tags inclusive ?

Reply:
modifySql